recipes-nodes/config-parser/render-confg.bb

```diff
@@ -0,0 +1,46 @@
+SUMMARY = "Mustache templates renderer wrapper"
+DESCRIPTION = "Render a mustache template using the provided JSON input configuration"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+FILESEXTRAPATHS:prepend := "${THISDIR}/:"
+SRC_URI = file://render-config.sh"
+
+S = "${WORKDIR}"
+
+python () {
+    # Check if INIT_CONFIG_URL is set in the environment or in local.conf
+    init_config_url = d.getVar('INIT_CONFIG_URL')
+
+    if init_config_url is None:
+        # If not set, check the original environment
+        origenv = d.getVar("BB_ORIGENV", False)
+        if origenv:
+            init_config_url = origenv.getVar('INIT_CONFIG_URL')
+
+    if init_config_url:
+        # If INIT_CONFIG_URL is set (to any non-empty value), keep its value
+        d.setVar('INIT_CONFIG_URL', init_config_url)
```

Comment:
This probably should be checking if it is not set and either:
- define a default URL where to fetch the config from. Or
- cause the build to crash with bb.fatal message specifying that it should be set.

Also, we should add that env variable to the yocto_build_confiog.env file in the yocto-manifest repo to keep track of all possible env variables that we offer